### ISSUE TITLE
Service Tiles - Link Should Be Learn More

### ISSUE THREAD

**Issue by colinvlasak** [aemsites/stericycle-shared#335]
Service Tiles – Url Should be Learn More Link not a full URL

![image](https://github.com/user-attachments/assets/88da34a2-25dd-406a-95af-3d9926a890b2)

Should be 

![image](https://github.com/user-attachments/assets/225846b8-d6e5-4780-b2b7-6bfda6792ecf)

**Comment by MarquiseRosier** [comment]
Fixsed

**Comment by colinvlasak** [comment]
@MarquiseRosier looks to be fixed on most pages however it hasn't updated on - https://main--shredit--stericycle.aem.live/en-us/secure-shredding-services/hard-drive-destruction

Can you see that this is fixed also can you confirm why it didn't update on this page?

![image](https://github.com/user-attachments/assets/65f0c241-3eb3-4d8a-80cc-a6e36e1a34fe)

**Comment by jindaliiita** [comment]
@colinvlasak This page was not published, so you were seeing the old content: https://main--shredit--stericycle.aem.page/en-us/secure-shredding-services/hard-drive-destruction. I have now published the page. In the future, if you encounter this issue, simply publish the page on the .page domain, like this: https://main--shredit--stericycle.aem.page/en-us/secure-shredding-services/hard-drive-destruction.
Closing this issue as of now.
